_includes/manuals/2.0/2.1_quickstart_installation.md

```diff
@@ -162,13 +162,12 @@ sudo foreman-installer
 {% endhighlight %}
 </div>
 
-After it completes, the installer will print some details about where to find Foreman and the Smart Proxy and Puppet master if they were installed along Foreman. Output should be similar to this:
+After it completes, the installer will print some details about where to find Foreman and the Smart Proxy. Output should be similar to this:
 
 {% highlight bash %}
   * Foreman is running at https://theforeman.example.com
       Initial credentials are admin / 3ekw5xtyXCoXxS29
   * Foreman Proxy is running at https://theforeman.example.com:8443
-  * Puppetmaster is running at port 8140
```

Comment:
This line was actually dropped in 2.0 as part of https://github.com/theforeman/foreman-installer/pull/436 so I went back to update it in 2.0-2.2 manuals as well